### PR TITLE
Remove false warning from duplicated data.

### PR DESCRIPTION
**Pull request by joakim-hove**
**Task**
Code produced false warnings of duplicated data when plotting. Do *not* understand what was wrong with the existing code, but this at least removes the false warning. 

Will backport this to stable.

**Pre un-WIP checklist**
- [x] Statoil tests pass locally
- [x] Have completed graphical integration test steps

### CODE REVIEW

**Comment by andreabrambilla** [comment]
what about 
```
if any(data.duplicated()):
  print "warning ..."
```

File: python/python/ert_gui/plottery/plot_data_gatherer.py [hunk 1]
```diff
@@ -72,11 +72,16 @@ def gatherSummaryData(ert, case, key):
         if not data.empty:
             data = data.reset_index()
 
-            if True in data.duplicated():
```
Comment (markusdregi) [inline python/python/ert_gui/plottery/plot_data_gatherer.py]: I think the correct syntax is `True in data.duplicated().values`or even better `data.duplicated().any()`

Comment (markusdregi) [reply python/python/ert_gui/plottery/plot_data_gatherer.py]: Did not see Andrea's comment. We should us a form of `any`. I think the current code looks for `True` in the indices.

Comment (andreabrambilla) [reply python/python/ert_gui/plottery/plot_data_gatherer.py]: yes, I have just checked, doing `'whatever' in data.duplicated()` would check `whatever` against the labels, not against the boolean flag

File: python/python/ert_gui/plottery/plot_data_gatherer.py [hunk 2]
```diff
@@ -72,11 +72,12 @@ def gatherSummaryData(ert, case, key):
...
-              print("** Warning: The simulation data contains duplicate "
-                    "timestamps. A possible explanation is that your "
-                    "simulation timestep is less than a second.")
-              data = data.drop_duplicates()
+            if True in any(data.dupliated()):
```
Comment (markusdregi) [inline python/python/ert_gui/plottery/plot_data_gatherer.py]: This should be 
`if any(data.dupliated()):`

Comment (joakim-hove) [reply python/python/ert_gui/plottery/plot_data_gatherer.py]: Thank you.

**Pull request merged**